backend/src/graphql_api/plt.rs

```diff
@@ -537,14 +537,19 @@ impl QueryPltAccountAmount {
                   OR ($2 = $3 AND COALESCE(pa.amount, 0) = $2 AND pa.account_index < $5 AND \
              pa.account_index > $4)
               )
-            ORDER BY COALESCE(pa.amount, 0) DESC, pa.account_index DESC
+            ORDER BY 
+                CASE WHEN $7 THEN COALESCE(pa.amount, 0) END ASC,
+                CASE WHEN $7 THEN pa.account_index END ASC,
+                CASE WHEN NOT $7 THEN COALESCE(pa.amount, 0) END DESC,
+                CASE WHEN NOT $7 THEN pa.account_index END DESC
```

Comment:
I was wondering if this query can be simplified? As I understand , all we are doing here is, if $7 = true sort both these fields in asc order other wise sort them in desc order.
 
If so could we just mutiply pa.amount &  pa.account_index with 1 for asc order and -1 for dec, probably this would make the query less verbose? 

Does it make sense?

Reply:
Checked here https://github.com/Concordium/concordium-scan/blob/370b8fe0e58390e7b4ff8336556c931da66af206/backend/src/graphql_api/baker.rs#L1251 its is done they way you have done, so better not change anything.